test/integration/image-component/no-intersection-observer-fallback/pages/_document.js

```diff
@@ -0,0 +1,19 @@
+import Document, { Html, Head, Main, NextScript } from 'next/document'
+
+export default class MyDocument extends Document {
+  render() {
+    return (
+      <Html>
+        <Head crossOrigin="anonymous">
+          <script
+            dangerouslySetInnerHTML={{ __html: 'IntersectionObserver = null' }}
+          />
```

Comment:
re https://github.com/vercel/next.js/pull/37447#discussion_r890734876 @styfle 

I have added a dedicated integration test case `test/integration/image-component/no-intersection-observer-fallback`. And I add `IntersectionObserver = null` to the top of the component. It should make sure the timing